src/protocols/aeternity/helpers/index.ts

```diff
@@ -124,6 +125,17 @@ export function categorizeContractCallTxObject(transaction: ITransaction): {
   }
 }
 
+export function convertMultisigAccountToAccount(
+  multisigAccount: IMultisigAccount,
+): Partial<IAccount> {
+  return {
+    address: multisigAccount.gaAccountId,
+    idx: 0,
+    protocol: PROTOCOL_AETERNITY,
+    globalIdx: 0,
```

Comment:
Why will they always have idx equal to 0?

Reply:
Good point. The reason is that we don't use indexes when displaying the multisig accounts. For example here are is the card:
![image](https://github.com/aeternity/superhero-wallet/assets/8217670/352c884c-5f81-4a97-9372-3f40a27c41dd)
I'll add a comment about the reason why both idx and globalIdx are equal to 0.
What do you think?

Reply:
I agree. It's always better to have an explanation